plugins/woocommerce-admin/client/homescreen/activity-panel/reviews/test/index.js

```diff
@@ -109,7 +109,7 @@ describe( 'ReviewsPanel', () => {
 			expect( screen.queryByText( 'Delete' ) ).toBeInTheDocument();
 		} );
 
-		it( 'should trigger updateReview with status approved when Approve is clicked', () => {
```

Comment:
I believe these are the tests that were disabled because of the dependency changes? We may want to create another issue to explore reenabling these, or understanding a better way to approach it.

Reply:
We can find the issue [here](https://github.com/woocommerce/woocommerce/issues/37088). Thanks